diff --git a/src/common/gyre_constants.fpp b/src/common/gyre_constants.fpp
index 10aead99..5f66ded2 100644
--- a/src/common/gyre_constants.fpp
+++ b/src/common/gyre_constants.fpp
@@ -61,7 +61,7 @@ module gyre_constants
 
   ! Paths
 
-  character(FILENAME_LEN), save, protected :: GYRE_DIR = ''
+  character(FILENAME_LEN), save, protected :: GYRE_DIR = 'NIX_GYRE_DIR'
 
   ! Interfaces
 
@@ -121,14 +121,9 @@ contains
 
     ! Read constants
 
-    call get_env('GYRE_DIR', gyre_dir_, status)
-    if (status == 0) GYRE_DIR = gyre_dir_
-      
     rewind(unit)
     read(unit, NML=constants)
 
-    $ASSERT(GYRE_DIR /= '',GYRE_DIR is not set)
-
     ! Finish
 
     return
